tests/test_chebyMap.py

```diff
@@ -491,6 +489,38 @@ def test_normalize(self):
             self.assertAlmostEqual(normAxis.min(), -1)
             self.assertAlmostEqual(normAxis.max(), 1)
 
+    def test_ChebyMapDM10496(self):
+        """Test for a segfault when simplifying a SeriesMap
+
+        We saw an intermittent segfault when simplifying a SeriesMap
+        consisting of the inverse of PolyMap with 2 inputs and one output
+        followed by its inverse (which should simplify to a UnitMap
+        with one input and one output). David Berry fixed this bug in AST
+        2017-05-10.
+
+        I tried this test on an older version of astshim and found that it
+        triggering a segfault nearly every time.
+        """
+        coeff_f = np.array([
+            [-1.1, 1, 2, 0],
+            [1.3, 1, 3, 1],
+        ])
+        coeff_i = np.array([
+            [1.6, 1, 3],
+            [-3.6, 2, 1],
+        ])
+        lbnd_f = [-2.0, -2.5]
+        ubnd_f = [1.5, -0.5]
+        lbnd_i = [-3.0]
+        ubnd_i = [-1.0]
+
+        for i in range(1000):
```

Comment:
Yes to both. Note: I originally had the assert in the loop, but found it greatly reduced the odds of a segfault, so I moved it out. However, I then found that creating `amap` in the loop greatly increased the odds of a segfault. It turns out that this remains so even with the assert in the loop, so I moved the assert back into the loop as you requested.